.env-sample

```diff
@@ -0,0 +1 @@
+JSON_SECRET=anysecretgoeshere
```

Comment:
an appropriate name would be `AUTH_SECRET` or `JWT_SECRET` cos `JSON_SECRET` sounds abit confusing, don't you think ?

Reply:
Thanks